api/src/org/apache/cloudstack/api/command/user/template/CreateTemplateCmd.java

```diff
@@ -124,6 +127,9 @@
                description = "true if template contains XS/VMWare tools inorder to support dynamic scaling of VM cpu/memory")
     protected Boolean isDynamicallyScalable;
 
+    @Parameter(name = ApiConstants.PROJECT_ID, type = CommandType.UUID, entityType = ProjectResponse.class, description = "create template for the project")
+    private Long projectId;
```

Comment:
Ping. Since, this is an append only API change. IMO we can merge on 4.5 as well.
Updates, is this good to go?